contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,750 @@
+        </tr>
+        <tr>
+            <td><strong>User paths</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track user flows and where they drop-off</td>
+        </tr>
+        <tr>
+            <td><strong>Retention</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Analyze user and revenue retention</td>
+        </tr>
+        <tr>
+            <td><strong>UTM tracking</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track marketing campaigns with UTM tags</td>
+        </tr>
+        <tr>
+            <td><strong>Correlation analysis</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Suggested events and properties that lead to success or failure</td>
+        </tr>
+        <tr>
+            <td><strong>Group analytics</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Understand how organizations use your product</td>
+        </tr>
+        <tr>
+            <td><strong>Formulas</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Use custom formulas to calculate unique insights</td>
+        </tr>
+        <tr>
+            <td><strong>Query editor</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Coming Q1 2023</td>
+            <td>Write your own queries in SQL</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+-   **Correlation analysis:** This feature enables you to [automatically find correlated events or properties](/manual/correlation) which affect the conversion rate of users within a funnel. LogRocket doesn’t offer any such automated correlation discovery, meaning users must search for correlating factors manually and without assistance.
+
+-   **Query editor:** LogRocket doesn’t currently support any way for users to interrogate data directly, either through SQL or other, abstracted query language. PostHog also doesn’t offer SQL queries…but [we’re actively working on it](/roadmap).
```

Comment:
```suggestion
-   **Query editor:** LogRocket doesn’t currently support any way for users to interrogate data directly, either through SQL or other, abstracted query language. PostHog also doesn’t offer SQL queries... but [we’re actively working on it](/roadmap).
```

I guess you're using Google Docs? Can you turn-off auto ellipsis? It renders weird online.